### DISCUSSION TITLE
Student free account

### DISCUSSION THREAD

**Discussion by SosoPataraia** [community/community#147104] [GitHub Education]
Hello, I was wondering how I can register as a student and get the student pack for free. The problem I'm facing is it requires me to upload a photo with the camera and my camera takes photos of quality more than 1MB which is restricted. Therefore I can't. 

Please help.

Thank you very much!

**Comment by TzivyaFireman** [comment]
Hey,
You can try using an online compression tool to reduce the image size to under 1MB.
For example, [TinyPNG](https://tinypng.com/) or [Compress JPEG](https://compressjpeg.com/).
Upload your image, let the tool compress it, and then download the resized version.
Hope this helps!

**Reply by SosoPataraia** [reply]
Yes, but it requires me to take a photo at the moment...

**Comment by TzivyaFireman** [comment]
Hey @SosoPataraia, I'd love to know if I was able to help you.
